### ISSUE TITLE
Add async read + write

### ISSUE THREAD

**Issue by LuMiSxh** [lise-henry/epub-builder#44]
Hey,
I'm creating multiple ePubs at once and thought about using async rust for file I/O but realized that this crate doesn't support async read and write.

Would it even be feasible sense to add that?

If so, I'd try implementing it myself.

**Comment by duyk471** [comment]
I guess it's not really feasible. Refactor the whole codebase just for that isn't worth it. Leveraging the power of async for simple tasks like generating some epub files is somewhat overkill. I guess I should ask @crowdagger and here is just my two cents.

**Comment by crowdagger** [comment]
Yeah I pretty much agree, particularly as it's not a domain I'm competent which would be a problem to maintain it :s
